src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx

```diff
@@ -75,18 +75,21 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const Cell: FC<{
-  cell: SurveyOptionsViewCell;
+  cell: SurveyOptionsViewCell | undefined;
 }> = ({ cell }) => {
   const { orgId } = useRouter().query;
   const styles = useStyles();
   const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
   const { openPane } = usePanes();
 
+  if (!cell) {
+    return null;
+  }
```

Comment:
Instead of adding a separate if statement like this, the next one could be `if (cell?.length == 0)`, thereby catching if `cell` is falsy, or if it has zero length.

Reply:
ahhh so much nicer! I´ll change it asap